auto-cert-kit.service

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=Service to run auto cert kit test
+Requires=xapi.service
+After=xapi.service
+
+[Service]
+Type=simple
+ExecStartPre=/opt/xensource/bin/xapi-wait-init-complete 300
```

Comment:
I understand the timeout added here is to make the service is not waiting xapi start always and blocked here without any time limit. So I have a question that if xapi-wait-init-complete exec timeout after 300, will ExecStart script still be executing.